### ISSUE TITLE
OPS Workflow Modifications for Inclusion of Moderator (Editor) Role 

### ISSUE THREAD

**Issue by willinsky** [pkp/pkp-lib#5622]
**Describe the bug**
OPS 

1. On setting up an OPS server, a Preprint Server Manager (PSM) is assigned. 
a. The PSM can sign in as anyone, unpost preprints, and establish categories. 
b. The PSM can set up categories appropriate for the OPS server, as the suggestion, for example, of the sponsoring organization. 

2. The “editor” role in OPS could be identified as a “moderator,” following the arXiv.org model of [moderation](https://arxiv.org/help/moderation). The OPS Manager could enroll as a moderator to then assign other moderators to “categories” of submissions for automatic notification on a request to post a preprint and for which the moderator would conduct the basic screening (see moderation link) and to then post the preprint. 

The author, after registering, sees the dashboard:
On the dashboard, “My Queue” stays the same, but “My Assigned” changes to “Papers” and “Archives” changes to “Preprints.” 
For a New Submission, under 1. Start, I would argue that Categories (by disciplines and subdisciplines) is the required, as it determines which moderators are automatically assigned to screen the paper, while Series is optional, and to which authors typically need to be invited (although I can see that’s too complicated at this point).  
In the “Upload submission” stage, the authors sees “Create New Galley” and “Galley Label” We need to replace our current note “Typically used to identify the file format (e.g. PDF, HTML, etc.)” with one like this: “The galley is the file that displays the preprint, and should be identified under Galley Label as a file type (e.g., PDF, HTML, Word).” We should remove from the pop-up “This galley will be available at a separate website” as it will only confuse authors.
After completing the submission, an author can then see a single tab “Publication” which we relabel as “Preprint.” It includes Preprint Discussions, for whom the default Participants are the assigned Moderator and the OPS Manager, with an option to invite others into the discussion of the preprint (which is not currently the case). Under Status, the two states are “Awaiting screening” and “Posted.” [If we include an “unmoderated” option, then the submission would go directly to Posted]

The moderator is assigned to Categories, created by the OPS Manager. The moderator is notified with each assigned submission, with a link leading to the author’s submission, under the Preprint tab, with an additional tab “Post” as in | Preprint | Post |. There is no need for an additional tab under Post (currently called “Production”)
After reviewing the content of the Preprint tab, the moderator goes to the Post tab, where there is a Post Discussions, where the moderator can engage with the author, and/or bring in the OPS Manager and/or other experts. 
There is a Participant list to which the moderator can add another moderator. Again, I don’t think that series editors have a place here. A series can be something an author can join on submitting the paper, and should be included in the metadata so that it can be altered.  
There is a button that should now read “Post the preprint.”

**Comment by alexxxmendonca** [comment]
> I would argue that Categories (by disciplines and sub-disciplines) is the required field for all authors, and should be used to determines which moderators are automatically assigned to screen the paper (while Series is optional for authors to select).

At SciELO we are actually viewing it differently. We are approaching Series as the major areas of knowledge (at SciELO they are 8), for instance, HEALTH SCIENCES, and we are approaching categories as sub-areas, for instance, Nursing.

We could use categories instead of series, but we were under the impression that we could only have different policies (author instructions) depending on the series. Can we have specific policies based on categories, too?

I pretty much agree with everything else suggested.

**Comment by asmecher** [comment]
I'd suggest keeping wording tweaks (e.g. relating to `Create New Galley`, and everything in `4` and `5`) in https://github.com/pkp/pkp-lib/issues/5610.

There are some good ideas here but they need to be aligned with the design work we've already done e.g. on screening plugins. The default screening behaviour, for example, is to require an Editor [we can rename this to Moderator but I'll stick with Editor for now] to manually post a preprint, roughly along the lines of what you've described. The addition of screening plugins (like @ajnyga's [`returningAuthorScreening`](https://github.com/ajnyga/returningAuthorScreening) plugin (previously called `defaultScreening`).

We would need to resolve a problem before Categories to define workflow behaviour (including automatic editor assignments), as discussed during our last meeting and as raised by @alexxxmendonca -- there can only be a single Series assigned, but there may be multiple Categories. So if two categories with conflicting workflow implications are assigned, we would need to determine how that is resolved. In my opinion, that's a defining distinction between Series/Sections and Categories -- Series/Sections are allowed to have workflow implications, but Categories (I propose) should not.

I think we're jumping through two stages at once here -- requirements and design. I'd suggest describing the Moderator role using "needs to" type requirement statements rather than describing the UI they'll work with to satisfy those needs.

**Comment by alexxxmendonca** [comment]
> Series/Sections are allowed to have workflow implications, but Categories (I propose) should not.

I agree with this and it is how I see it too.

For instance, and please correct me if I'm wrong, different Series would allow for different policies (much like Section Policy in OJS), but that's not the case for Categories.

**Comment by willinsky** [comment]
One way to approach the terms at issue (if not the current function) with "categories" and "series" (and sections), is by drawing on the example of the leading preprint servers. Three of the key players --  [arXiv.org](https://arxiv.org/about), [biorxiv.org](https://www.biorxiv.org/) and [SSRN](https://www-ssrn-com.stanford.idm.oclc.org/index.cfm/en/) -- each use different terms -- fields, subject areas, and networks, respectively -- for how all of the content on the server is divided into one of what could also be called disciplines or topics (e.g., architecture, physics, biochemistry). Each server has their own limited set of topics, and for readers browsing, this approach make great sense. Only arXiv.org, the oldest, has "categories" within "fields" (e.g., machine learning within Stats), as well as allowing for "cross-lists" that place a paper in two categories. All three allow you to browse by field, subject area, and network, respectively. 

While I find "categories" closer to this discipline-based approach than "series," the three main servers make it clear that there are no standard terms. Still, I'd propose that the server manager (perhaps on the advice of the Advisory Committee) set out the server's "categories" (or some other term for this sort of thing), with authors having to choose one or two (if possible). 

Keywords can be used to designate type of study, such as review, clinical triaL And I see "series" as a more advanced feature for down the road, to be used to designate and group a set of papers from a conference or a lab, with the series something an author could ask a manager to set up for a specific list of authors in the server.  I do hope that this might advance the work on this aspect of OPS.

**Comment by asmecher** [comment]
@ajnyga, @NateWr, and I have been chatting about elements of this and carving off pieces for implementation:
- **Sections:** @ajnyga prefers "sections" rather than "series" and I agree with him. This isn't as clear an organizational tool in OPS as "section" is in OJS or "series" in OMP, but I continue to see value in it. The most common use case I foresee will be as a kind of content type (like "review article" and "research article" in OJS), but there are other possible uses, e.g. distinguishing between multiple labs. When only one section is created (which is the default state of OPS), the "sections" dropdown in various parts of the system will be hidden, making this an unobtrusive option for those who want it.
- **Categories and automatic assignments:** We're adding support for associations between categories and "sub editors" (this is the generic term we're using in the code for "series editors" in OMP and "section editors" in OJS, and what you're calling a "moderator" here, John). When a preprint is submitted, the author can choose from a list of categories. OPS will automatically assign *all* assigned sub editors for *all* assigned categories. I'm not 100% sure that's the best approach, but unsure how to improve on it.

**Comment by willinsky** [comment]
I agree with sections (series are thematic and usually run by their own editors as a separate project). SSRN uses sections, with one for conference papers, for example, although one for research paper series, as well. But their big division by subject area or discipline is networks and ours would be categories with moderators automatically assigned. I can see the issue with all assigned but again we have some time before that becomes an issue to be handled by moderators-in-chief or some such, just as we might later bring in a way for an author or group to organize their own lab series.

**Comment by ajnyga** [comment]
thanks @asmecher and @willinsky 
I have the code changes pretty much ready for what Alec described above.

I will make the OPS Default Theme "Category centric". Meaning that browsing and searching of Preprints is mainly organized around Categories. Of course any user of OPS can make their own theme more "Section centric" if that taxonomy makes more sense to them. 

The code for Category browsing comes from the shared library so I can see a lot of value in developing the feature further. It will benefit OJS as well for users that want to publish without issues.

**Comment by alexxxmendonca** [comment]
I agree that "sections" works better than "series".
